packages/rocketchat-authorization/server/startup.coffee

```diff
@@ -7,7 +7,8 @@ Meteor.startup ->
 	permissions = [
 		{ _id: 'access-permissions',            roles : ['admin'] }
 		{ _id: 'add-oauth-service',             roles : ['admin'] }
-		{ _id: 'add-user-to-room',              roles : ['admin', 'owner', 'moderator'] }
+		{ _id: 'add-user-to-own-room',          roles : ['admin', 'owner', 'moderator'] }
+    { _id: 'add-user-to-any-room',          roles : ['admin'] }
```

Comment:
Oh man, this always happens with my commits, I expect RC to pick up .editorconfig but for some reason it isn't. Thanks for catching.